Modules/CaPTkCommon/src/CaPTkROIConstructionImplementation.hxx

```diff
@@ -270,7 +270,7 @@ void captk::ROIConstructionImplementation< TImage >::Update()
     // if both the latticeWindow and latticeStep are valid sizes, proceed with grid construction
     if (m_latticeEnabled)
     {
-      std::cerr << ("Found valid latticeStep and latticeWindowSize; lattice-based computation has been enabled");
+      std::cout << "Found valid latticeStep and latticeWindowSize; lattice-based computation has been enabled\n";
```

Comment:
Use of MITK_INFO?

Reply:
CaPTkROIConstructionImplementation.* is CaPTk's code. I think it's better to not MITKify it. But if you want I can change it. let me know

Reply:
where will the std::cout show in case of gui usage? 
Also we cant be writing our own logger. better to use mitk logger.

Reply:
(please see my comment below in response to this)